### ISSUE TITLE
[FEATURE] No BR

### ISSUE THREAD

**Issue by kopax-polyconseil** [Riverside-Healthcare/djLint#571]
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Feature Request

Add a rule to forbid the usage of `<br />`, instead use styling

Thanks! 🤠

**Comment by kopax-polyconseil** [comment]
My team complained about the inhability to configure djlint to not reformat when line is short, and to see randomly lines being reformated because something change elsewhere, which screw the diff.

I believe this should be the first addressed issues. But this seems to be a dead project, isn't it ? 

Thanks for sharing !

**Comment by kopax-polyconseil** [comment]
I have many, I just had to check in my recent PR where HTML gets edited: https://github.com/pass-culture/pass-culture-main/pull/5862/files#diff-c1c3cba64c7581d62a79c950ed5a18861eb411c7c610fef1a2a44faa4be50b6bR126

**Comment by christopherpickering** [comment]
I'll take a look shortly. One thing you can try - if you want all lines to be broken and indented, set the max line length to 0.

What output are you expecting from djlint that you are not getting?

That will force stuff like 
```html
<td>hi</td>
```
to become
```html
<td>
    hi
</td>
```
https://djlint.com/demo/
![image](https://user-images.githubusercontent.com/17788706/229138601-3c07b189-cde1-4c62-9e54-93ea4359ee7d.png)

**Comment by kopax-polyconseil** [comment]
We have max line length 120: https://github.com/pass-culture/pass-culture-main/blob/master/api/src/pcapi/routes/backoffice_v3/.djlintrc#L8

With the following HTML (from the demo):

```html
<div>
    <p>Welcome to djLint online!</p>
</div>
```

`    <p>Welcome to djLint online!</p>`.length = 36

I expect if I set the max line length to 35, to have the line reformated, but it doesn't work.

**Comment by christopherpickering** [comment]
That calculation could be improved.

**Comment by kopax-polyconseil** [comment]
> That calculation could be improved. Could you do a pr to fix it?

By `improve` you mean `fixed` ? I'll be glad to help if I knew, but I am not sure where to do that.

**Comment by christopherpickering** [comment]
Thanks! 
for now you could try a configuration like this and see how it works for you.

```json
{
  "profile": "jinja",
  "extension": "html",
  "reformat": true,
  "format": true,
  "indent": 2,
  "preserve_leading_space": false,
  "max_line_length": 0,
  "max_attribute_length": 30,
  "ignore": "H006",
  "use_gitignore": true,
  "format_js": true,
  "js": {
    "indent_size": 2
  },
  "format_css": true,
  "css": {
    "indent_size": 2
  }
}
```

Setting the max line length to 0 will make tags always wrap when possible (some tags cannot wrap). Leaving it at 120 will let djlint not wrap shorter lines.

I'm not sure how else I can help you... if you have examples of incorrect formatting exactly, maybe I could look at them.  Your diff looks pretty good to me.

I'm free Monday AM Chicago time if you'd like to call on discord.

**Comment by kopax-polyconseil** [comment]
> Monday AM Chicago time

What exact time are you reffering to ? We are in CET tz. 

> for now you could try a configuration like this and see how it works for you.

I'll check with my team, but I already know they want to be able to keep things on one line when the line is short.

**Comment by christopherpickering** [comment]
which file was that in?

**Comment by christopherpickering** [comment]
> > Monday AM Chicago time
> 
> What exact time are you reffering to ? We are in CET tz.
> 

Sometime between 9 AM CST and 12 CST. Just send a discord message and I'll jump on.

**Comment by christopherpickering** [comment]
Cool sounds good, no problem! https://discord.gg/taghAqebzU

**Comment by christopherpickering** [comment]
@kopax-polyconseil I opened issue #580 for the line length problems, we can continue that chat there. Thank you!

**Comment by christopherpickering** [comment]
@kopax-polyconseil The next release will have a rule H036 for finding <br> tags.

The rule is disabled by default, but you can include it with `--include=H036`, or in the config file `include="H036"`

Thanks!
